contents/blog/clickhouse-vs-redshift.md

```diff
@@ -0,0 +1,150 @@
+6. The leader node returns that value to the BI tool. 
+
+Things get extra complicated, and extra efficient, with Amazon’s 2019 release of AQUA (Advanced Query Accelerator). AQUA evolved because SSD bandwidth increased immensely over time, but more storage doesn’t inherently mean faster queries, especially when the bottleneck is now between the memory-compute bus. 
+
+AQUA is a hardware optimization that enables storage units to run computations without burdening the main compute layers (e.g., Redshift Spectrum). AQUA is **not** a replacement for Spectrum, it just eases the load by handling household tasks, such as encryption or compression, as well as precomputing simple aggregations of data. 
+
+According to AWS, AQUA boosts performance by 10x on average by precomputing queries. 
+
+![AQUA.png](../images/blog/clickhouse-vs-redshift/AQUA.png)
+
+AQUA shifts Amazon’s shared-nothing model to something similar to a shared storage model. Because AQUA precomputes data, it encourages developers to use a single Amazon Redshift managed storage layer that multiple Redshift clusters can interact with. 
+
+### ClickHouse’s (traditional) architecture
+
+ClickHouse’s traditional architecture does differ from some managed instances of ClickHouse, but it best represents ClickHouse’s design differences. 
+
+ClickHouse extends a shared-nothing architecture, combining CPU, storage, and memory into a single, beefy machine. But while ClickHouse’s design is fundamentally monolithic, it also supports sharding a database across multiple instances using Apache ZooKeeper. 
+
+![Shared Nothing Architecture.png](../images/blog/clickhouse-vs-redshift/Shared_Nothing_Architecture.png)
+
+Because ClickHouse bundles everything into the same instance, it can scale that instance into three discrete dimensions – storage, memory, and compute. Scaling an already provisioned instance isn’t trivial; a new instance needs to be spun up and the data needs to be manually migrated. 
+
+![Clickhouse_ 3 Dimensions.png](../images/blog/clickhouse-vs-redshift/Clickhouse__3_Dimensions.png)
+
+Like Redshift, ClickHouse employs some techniques for pre-aggregating data to return faster queries. Redshift accomplishes this by specialized in-memory hardware that precomputes queries. ClickHouse, by contrast, uses materialized views with custom settings that dynamically and efficiently precompute data, even once new data is ingested. 
+
+In layman's terms ClickHouse is a speedboat whereas Amazon Redshift is an aircraft carrier – one is most definitely feature-packed (Redshift), while the other is faster (ClickHouse). And sometimes, that speed is the deciding factor.
+
+### ClickHouse Cloud’s architecture
+
+A common point of confusion is the distinction between ClickHouse and ClickHouse Cloud, particularly in regard to architecture. While ClickHouse is a monolithic titan, ClickHouse Cloud decouples storage and compute. It is deployed on AWS and, as of very recently, GCP. 
```

Comment:
Maybe we should do an article for ClickHouse vs ClickHouse Cloud?

Reply:
Possibly. Will have a think.